flake.nix

```diff
@@ -109,6 +109,7 @@
       lenovo-thinkpad-x260 = import ./lenovo/thinkpad/x260;
       lenovo-thinkpad-x270 = import ./lenovo/thinkpad/x270;
       lenovo-thinkpad-x280 = import ./lenovo/thinkpad/x280;
+      lenovo-thinkpad-z13 = import ./lenovo/thinkpad/z-series/z13;
```

Comment:
Suggestion:

```diff
- lenovo-thinkpad-z13 = import ./lenovo/thinkpad/z-series/z13;
+ lenovo-thinkpad-z-series = import ./lenovo/thinkpad/z-series;
+ lenovo-thinkpad-z-series-z13 = import ./lenovo/thinkpad/z-series/z13;
```

Though I could see an argument for dropping the `z-series-*` from the `z13` for brevity (though you'll likely ever only type this once in your config).

Reply:
None of the other Lenovo laptops mention `*-series`, should we consider just `lenovo-thinkpad-z` + `lenovo-thinkpad-z13`??